hw2/04-background.js

```diff
@@ -1 +1,59 @@
 // Add your code here
+const START_CLASS = "btn-primary";
+const STOP_CLASS = "btn-danger";
+const BASE_TRANSITION_DURATION = 500;
```

Comment:
I think the base transition duration asked in the assignment is 3 seconds.

Reply:
oh I totally missed that! Thanks for pointing this out 💯

Reply:
You are welcome!